crates/syntax/src/ast/make.rs

```diff
@@ -863,6 +863,36 @@ pub fn param_list(
     ast_from_text(&list)
 }
 
+pub fn trait_(
+    is_unsafe: bool,
+    ident: String,
```

Comment:
```suggestion
    ident: &str,
```

because I like removing allocations as much as possible :laughing: